FirebaseAdmin/FirebaseAdmin/Auth/Multitenancy/TenantManager.cs

```diff
@@ -336,8 +337,9 @@ void IDisposable.Dispose()
 
         internal static TenantManager Create(FirebaseApp app)
         {
-            var args = new Args
+            var args = new Args()
```

Comment:
Just wondering why was this required?

Reply:
It's just a cleaner and shorter syntax. We ought to use it whenever possible. Not a must though.